### PR TITLE
[MADPORT-431] Added a method to despawn all entities via a spawn ticket

### PR DESCRIPTION
**Pull request by vmednonogov**
## What does this PR do?

Added a method to despawn all entities via a spawn ticket

See also https://github.com/carbonated-dev/o3de-gruber/pull/314

## How was this PR tested?

Verified locally

### CODE REVIEW

File: Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp [hunk 1]
```diff
@@ -305,6 +308,45 @@ namespace AzFramework
         }
         return SpawnableInstanceDescriptor::GetInvalidDescriptor();
     }
+
+    void SpawnableEntitiesManager::DespawnAllEntitiesInTicketByEntityID(const AZ::EntityId& entityId, DespawnAllEntitiesOptionalArgs optionalArgs)
+    {
+        if (entityId.IsValid())
+        {
+            EntitySpawnTicket::Id ticketId = 0;
```
Comment (IVasilets) [inline Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp]: Maybe something like `const EntitySpawnTicket::Id InvalidTicketId = -1;` and use it as a named constant here and below

**Review by IVasilets — APPROVED** [review]
LGTM

**Review by VladimirLagutin — APPROVED** [review]
LGTM

File: Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp (hunk 1 above)
Comment (vmednonogov) [reply Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp]: Updated

**Pull request merged**